get_all_manifests.sh

```diff
@@ -6,20 +6,32 @@ GITHUB_URL="https://github.com"
 # component: notebook, dsp, kserve, dashbaord, cf/ray/kueue/trainingoperator, trustyai, modelmesh, modelregistry.
 # in the format of "repo-org:repo-name:ref-name:source-folder:target-folder".
```

Comment:
We can probably get rid of target-folder and make it always as the key?

Reply:
> We can probably get rid of target-folder and make it always as the key?

hmm, odh-notebook-controller :)

Reply:
https://github.com/opendatahub-io/opendatahub-operator/pull/1456

Reply:
Can we do #1456 after this one ?

The reason is that the component keys have also changed i.e. `workbenches` related components are cloned to a `workbenches` folder so we'll have:

- workbenches/kf-notebook-controller
- workbenches/odh-notebook-controller
- workbenches/notebooks

Also the path of the component has been changed to reflect the name of the actual component so i.e. there is `modelregistry` and not `model-registry-operator`